tests/v1alpha_project_test.go

```diff
@@ -104,7 +104,6 @@ func generateV1alphaProject(t *testing.T) v1alphaProject.Project {
 
 func assertV1alphaProjectsAreEqual(t *testing.T, expected, actual v1alphaProject.Project) {
 	t.Helper()
-	expected = deepCopyObject(t, expected)
```

Comment:
could we get rid of it in other places?

Reply:
I got rid of it there cause expected object was not modified, we need it in other places due to data races, these tests are run in parallel but they operate on a single object instance